python/utilities/log_list/print_log_list.py

```diff
@@ -8,7 +8,7 @@
 import sys
 import time
 
-from absl import flags as gflags
+from absl import app, flags as gflags
```

Comment:
I think we'd want those imports on separate lines.

Reply:
Done.